### PR TITLE
Fix types for cennzx to support 18dp balance

### PR DESCRIPTION
**Pull request by KarishmaBothara**
With this PR - CENNZX can support asset with 18 dp balance...
Using node at https://github.com/cennznet/cennznet/pull/456

### CODE REVIEW

**Comment by KarishmaBothara** [comment]
Screen Shot 2021-06-14 at 10 00 23 AM
All my test works fine locally... donno if I need a new docker image of cennznet node to run against...

File: packages/api/test/e2e/cennzx.e2e.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@
 import { Keyring } from '@polkadot/keyring';
 import { cryptoWaitReady } from '@polkadot/util-crypto';
 import initApiPromise from '../../../../jest/initApiPromise';
-import { Balance } from '@cennznet/types';
+import {Balance, LiquidityPriceResponse, LiquidityValueResponse} from '@cennznet/types';
+import {U256} from "@polkadot/types";
```
Comment (holygits) [inline packages/api/test/e2e/cennzx.e2e.ts:19]: nit: don't need `U256` anymore

File: packages/api/test/e2e/cennzx.e2e.ts [hunk 2]
```diff
@@ -69,10 +70,10 @@ describe('CENNZX RPC calls testing', () => {
     });
 
     it("Get the liquidity value for CENNZ asset in Alice's account", async done => {
-      const [liquidityVolume, coreValue, assetValue] = await api.rpc.cennzx.liquidityValue(alice.address, CENNZ);
-      expect(liquidityVolume.isZero()).toBe(false);
-      expect(coreValue.isZero()).toBe(false);
-      expect(assetValue.isZero()).toBe(false);
+      const liqudityValue: LiquidityValueResponse= await api.rpc.cennzx.liquidityValue(alice.address, CENNZ);
```
Comment (holygits) [inline packages/api/test/e2e/cennzx.e2e.ts:73]:
```suggestion
      const liqudityValue: LiquidityValueResponse = await api.rpc.cennzx.liquidityValue(alice.address, CENNZ);
```

**Review by holygits — COMMENTED** [review]
Can we add a test for 18dp asset query?

**Review by eupston — APPROVED** [review]
lgtm. Strange tests don't pass with 1.5.0 image. What image version are you running locally?

**Comment by holygits** [comment]
we'll release a 1.5.1 patch image once the cennznet node PR is cleared

**Comment by KarishmaBothara** [comment]
> Can we add a test for 18dp asset query?

Added test for 18dp asset query...

**Review by JasonTulp — APPROVED** [review]
lgtm

**Pull request merged**